src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -95,15 +95,15 @@ trait MethodConfigurationService extends HttpService with PerRequestCreator with
         } ~ pathPrefix(Segment / Segment) { (configNamespace, configName) =>
           pathEnd {
             passthrough(
-              MethodConfigurationService.remoteMethodConfigUrl(workspaceNamespace, workspaceName, configNamespace, configName),
+              encodeUri(MethodConfigurationService.remoteMethodConfigUrl(workspaceNamespace, workspaceName, configNamespace, configName)),
               HttpMethods.GET, HttpMethods.PUT, HttpMethods.POST, HttpMethods.DELETE)
           } ~
           path("rename") {
-            passthrough(MethodConfigurationService.remoteMethodConfigRenameUrl(workspaceNamespace, workspaceName, configNamespace, configName),
+            passthrough(encodeUri(MethodConfigurationService.remoteMethodConfigRenameUrl(workspaceNamespace, workspaceName, configNamespace, configName)),
```

Comment:
would it be bad to put encodeUri within passthrough?

Reply:
i thought about doing that but didn't want to apply it to dozens of other passthroughs in case there were unexpected consequences, but that fear is probably unfounded. we'd still need to encode all of the URLs used by the DAO